src/renderercommon/tr_common.h

```diff
@@ -269,4 +269,11 @@ void R_LoadPNG(const char *name, byte **pic, int *width, int *height, byte alpha
 void R_LoadTGA(const char *name, byte **pic, int *width, int *height, byte alphaByte);
 void R_LoadSVG(const char *name, byte **pic, int *width, int *height, byte alphaByte);
 
+/*
+=============================================================
+IMAGE WRITERS
+=============================================================
+*/
+void RE_SaveTGA(const char *filename, byte *data, int width, int height, qboolean withAlpha);
```

Comment:
Does not seem to be used?

Reply:
Was used by the tr_font.c as means to output the generate font canvas. Its commented out currently but I used it in debugging. Not hurting to have it declared on the header tho (for debugging).